eZ/Publish/Core/Repository/ContentTypeService.php

```diff
@@ -552,14 +552,6 @@ protected function validateInputContentTypeCreateStruct( APIContentTypeCreateStr
             );
         }
 
-        if ( empty( $contentTypeCreateStruct->fieldDefinitions ) )
```

Comment:
This removal goes against what is documented on the interface, so it is really an API change. So this needs updated interface docblocks and BC notes.